src/main/msp/msp.c

```diff
@@ -3961,23 +3960,43 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
                     textVar = currentControlRateProfile->profileName;
                     break;
 
+                case MSP2TEXT_CUSTOM_MSG:
```

Comment:
I don't see any advantage of passing all three messages together. Allocating 3  entries (or maybe more, 
 to make it easier to add more messages) after `MSP2TEXT_CUSTOM_MSG` will make code and protocol simpler.
```suggestion
                case MSP2TEXT_CUSTOM_MSG_0:
                case MSP2TEXT_CUSTOM_MSG_1:
                case MSP2TEXT_CUSTOM_MSG_2: {
                   unsigned msgIdx = textType - MSP2TEXT_CUSTOM_MSG_0;
```
Or, maybe use `case MSP2TEXT_CUSTOM_MSG .. MSP2TEXT_CUSTOM_MSG + MSP2TEXT_CUSTOM_MSG_COUNT - 1:`, but it is gcc/clang extension